eng/depProj.targets

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This looks the same as the file in CoreFX. Should we have some central place we share this, presumably Arcade?

In general is the eng folder here and in CoreFX a mixture of Arcade stuff and repo stuff, there's not separate folders?

Reply:
Right, we knew that there will be a few duplicates as we don't want to move everything into arcade. Maybe we can either mirror these files manually (does the mirror support single files?), find another place where they can live or just live with the duplicates for now.